.github/workflows/schedule.cli.yml

```diff
@@ -0,0 +1,38 @@
+name: Schedule cli
+on:
+  workflow_run:
+    workflows: ["Pre submits cli"]
+    types: [completed]
+    branches: [main]
+
+permissions: read-all
+
+env:
+  GH_TOKEN: ${{ secrets.CREATE_ISSUES }}
+  ISSUE_REPOSITORY: ${{ github.repository }}
+
+jobs:
+  if-failed:
+    runs-on: ubuntu-latest
+    # See https://github.com/orgs/community/discussions/26238.
+    if: github.event.workflow_run.conclusion != 'success'
+    steps:
+      - uses: actions/download-artifact@9782bd6a9848b53b110e712e20e42d89988822b7 # tag=v1.10.0
+        with:
+          name: event_name
+      - name: Check event name
+        id: name
+        run: |
+          name=$(cat ./event_name.txt)
+          ctned="false"
+          if [[ "$name" == "schedule" ]] || [[ "$name" == "workflow_dispatch" ]]; then
+            ctned="true"
+          fi
+          echo "continue=$ctned" >> $GITHUB_OUTPUT
+      - uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579 # tag=v2.4.0
+        if: steps.name.outputs.continue == 'true'
```

Comment:
nit: this is ok, but at some point we may want to just have steps like `name` fail and continue on failure instead of setting a `continue` output flag. That might be cleaner. I know we already use this pattern a lot in shims for our e2e tests.

Reply:
I can change it if you think it's cleaner? Not clear to me

Reply:
Yeah, this is fine for now. I'm mostly just musing about making this cleaner in the future.